.eslintrc.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
     parser: 'babel-eslint'
   },
   rules: {
-    'no-console': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
+    'no-console': ['warn', { allow: ['warn', 'error'] }],
```

Comment:
This means: allow `console.warn`and `console.error` in `catch` blocks. That is, do not show lint warnings in those cases. Note that now GitHub actions doesn't show those warnings in this PR anymore.

I also left such config enable regardless the env, do you agree with that? I don't see why to leave it enabled in production but not dev.